app/views/devise/sessions/new.html.slim

```diff
@@ -1,25 +1,28 @@
-h2
-  | Log in
-
-= render 'shared/notifications'
```

Comment:
I agree. We can adjust the padding of the error messages and such after implementation. That makes it much easier to test. In the long run, we need to think about error handling in general and create a concept for that. However, that is for June and beyond, not March.